plugins/tutor-contrib-blockstore-minio/openedx-common-settings

```diff
@@ -0,0 +1 @@
+BUNDLE_ASSET_STORAGE_SETTINGS = dict(STORAGE_CLASS="django.core.files.storage.FileSystemStorage", STORAGE_KWARGS=dict(location="/openedx/media/blockstore/", base_url="http://localhost:8000/media/blockstore/"))
```

Comment:
Same as with the filesystem version: this should either be moved to `patches/` or removed.

Reply:
https://github.com/openedx/openedx-tutor-plugins/compare/5bc274b641ee38ed87655fe2329264cbe31f8945..e8d8951373df730585ba835127cd5b1f253250f2